controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -109,6 +111,7 @@ func newReconciler(mgr manager.Manager, ci hcoutil.ClusterInfo, upgradeableCond
 	}
 
 	r := &ReconcileHyperConverged{
+		config:               mgr.GetConfig(),
```

Comment:
couldn't use the client to make the necessary request to the` /nodes/proxy/configz` endpoint, so I added the config to the struct so that later can create a new base client